NitroxClient/GameLogic/RemotePlayer.cs

```diff
@@ -268,5 +270,19 @@ private void UpdateEquipmentVisibility()
         {
             playerModelManager.UpdateEquipmentVisibility(new ReadOnlyCollection<TechType>(equipment.ToList()));
         }
+
+        /// <summary>
+        /// Allows the remote player model to have its lightings dynamicly adjusted
+        /// </summary>
+        private void SetupSkyAppliers()
+        {
+            // SkyAppliers are the components that apply the light effects on gameobjects
+            SkyApplier skyApplier = Body.AddComponent<SkyApplier>();
+            skyApplier.anchorSky = Skies.Auto;
+            skyApplier.emissiveFromPower = false;
+            skyApplier.dynamic = true;
+            skyApplier.renderers = Body.GetComponentsInChildren<SkinnedMeshRenderer>(true);
+            Log.Debug($"Successfully setup Sky Appliers on {PlayerName}");
```

Comment:
Minor: I wouldn't use the log here. You aren't (and probably can't) check if the SkyAppliers is successfully setup. Therefore this log will show up independent from the success of the method, only exception is a hard throw which would be logged anyway.

Reply:
Maybe I can insert a try catch with a first call to the ``SkyApplier.ApplySkybox()`` method and print if it was successful or an error

Reply:
The question is do we expect the method to fail? If not we shouldn't try catch it imo.

Reply:
The ``SkyApplier.ApplySkybox()`` will fail if the skyApplier isn't correctly setup

Reply:
Looking at the code I don't see where it could fail as it checks for null for every variable used. Am I missing something?

Reply:
I guess it's because of the renderers part, when I first created the function, I didn't include the ``.renderers = ...`` part and for some reasons, errors were thrown and you couldn't even load into the game.

Reply:
That's true if `renderers` is null it can throw but now that you use `Body.GetComponentsInChildren<SkinnedMeshRenderer>(true)` it should never be null, only not filled with anything in the worst case.

Reply:
Ok, I'll just remove the logging then